types/oscal/profile/profile.go

```diff
@@ -55,20 +55,20 @@ type Modify struct {
 // catalog) being imported
 type Include struct {
 	All              *All    `xml:"all,omitempty" json:"all,omitempty"`
-	IdSelectors      []Call  `xml:"call,omitempty" json:"calls,omitempty"`
+	IdSelectors      []Call  `xml:"call,omitempty" json:"idSelectors,omitempty"`
 	PatternSelectors []Match `xml:"match,omitempty" json:"patternSelectors,omitempty"`
 }
 
 // Which controls and subcontrols to exclude from the resource (source catalog)
 // being imported
 type Exclude struct {
-	IdSelectors      []Call  `xml:"call,omitempty" json:"calls,omitempty"`
+	IdSelectors      []Call  `xml:"call,omitempty" json:"idSelectors,omitempty"`
 	PatternSelectors []Match `xml:"match,omitempty" json:"patternSelectors,omitempty"`
 }
 
 // A parameter setting, to be propagated to points of insertion
 type SetParam struct {
-	Id           string               `xml:"param-id,attr,omitempty" json:"id,omitempty"`
+	Id           string               `xml:"id,attr,omitempty" json:"id,omitempty"`
```

Comment:
I believe we had a discussion on this and you created a ticket for the upstream. Is this change by intention?

Reply:
This was re-generated from the profile metaschema which, as we've discussed, has a bug per https://github.com/usnistgov/OSCAL/issues/287. I'll revert these changes.

Reply:
awesome. other than this everything LGTM, Please go ahead and merge when you find time with these changes and i'll rebase the dependent PR

Reply:
Will do. Thanks